examples/image_classification-tf.ipynb

```diff
@@ -0,0 +1,7883 @@
+{
```

Comment:
<p>Will it make sense to also show how the users can use the model with their own explicit <code>tf.data</code> pipelines?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>

Reply:
<p>Yes - definitely! I think that work should be added in a separate PR (and possibly even notebook)</p>